scalardb-graphql-sample/README.md

```diff
@@ -0,0 +1,130 @@
+# Scalar DB GraphQL Sample
+
+This sample application is a simple CLI electronic money application that has the following features:
+
+- Charge an `amount` to a `user`
+- Pay an `amount` from a `user` to another `user`
+- Show the balance of a `user`
+
+In this sample, we will use Cassandra as a storage for Scalar DB. The Cassandra server and the Scalar DB GraphQL server will run with Docker.
+
+Apart from them, we will build a client Node.js application that will communicate with the GraphQL server. Please note that this client app is just one example of the clients; since GraphQL is a communication pattern, there are many tools in various programming languages for bulding applications.
+
+## Prerequisites
+
+- Docker and Docker Compose
```

Comment:
If you use a new feature of docker-compose, should we specify the version requirement?

Reply:
Added the versions for Docker Engine and Docker Compose.
https://docs.docker.com/engine/release-notes/
https://docs.docker.com/compose/release-notes/